Adding a Bedrock Example (#191)
Dinnerbone/mcstatus: README.md

print(f"The server has the following players online: {', '.join(query.players.names)}")
```

Bedrock Edition
```python
from mcstatus import MinecraftBedrockServer

> # Use MineCraftBedrockServer for Bedrock servers. If you do not specify a port, 19132 will be used by default.
> server = MinecraftBedrockServer(host="example.org", port=19132)

Reviewer: Show the `lookup` method as well as the default constructor, just like in the java version example
```suggestion
# If you know the host and port, you may skip this and use MinecraftBedrockServer("example.org", 19132)
server = MinecraftBedrockServer.lookup("example.org:19132")
```

# 'status' is the only feature that is supported by Bedrock at this time.

> # In this case status can also include motd, map, gamemode, and players_max. (ex: status.gamemode)

Reviewer: Fix the wording here, "can also include" is weird because it can include the listed things on top of what?
```suggestion
# In this case status includes motd, map, gamemode, and players_max. (ex: status.gamemode)
```

Author: I left it out because I used your examples in the code and did not include them in the comment. I added those to the comment and changed it to include. Great work on this project by the way. This has been very useful to me.
